provisions/roles/scanner/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Install Atomic
+  yum: name=atomic state=present
+  sudo: yes
+
+- name: Get Docker installed
+  service: name=docker enabled=yes state=started
+  sudo: yes
+
```

Comment:
Originally I started writing comment as I only read **Get Docker installed** whereas `atomic` already pulls in `docker` as a dependency. It's starting the service so maybe **Start and enable Doker** ?